packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -2794,6 +2797,9 @@ class DiagnosticableNode<T extends Diagnosticable> extends DiagnosticsNode {
 
   @override
   String toDescription({ TextTreeConfiguration parentConfiguration }) {
+    if (kReleaseMode) {
+      return toString();
```

Comment:
Why not make toDescription be the empty string? I would worry you could get a stack overflowing release mode for this case as "toString()" may call toDescription() within its implementation if you weren't elsewhere also short circuiting toString